plugin/trino-kinesis/src/test/java/io/trino/plugin/kinesis/TestRecordAccess.java

```diff
@@ -97,7 +100,7 @@ private void createDummyMessages(String streamName, int count)
         mockClient.putRecords(putRecordsRequest);
     }
 
-    private void createJsonMessages(String streamName, int count, int idStart)
+    private void createJsonMessages(String streamName, int count, int idStart, boolean compress)
```

Comment:
can we have utility methods below the usage (below test)? Please reorder methods in separate commit.